mautic/mautic
# Disabled send button If sending is in background

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:

If email is sending in background, then we need disable send button to avoid send duplicate emails. This happen already few times for customers.


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Setup Publish UP of segment email date lower like now
3.  Go to detail of email and see send button is disabled with tooltip message

![image](https://user-images.githubusercontent.com/462477/180605853-08ae7b3f-2f14-45f5-a66b-0cd7058b6223.png)





<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11351"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
